assets/script/object.js

```diff
@@ -0,0 +1,236 @@
+const projects = [
+  {
+    id: 'card1',
+    title: 'Tonic',
+    title1: 'Tonic',
+    company: 'CANOPY',
+    company1: 'CANOPY',
+    icon: './assets/icons/modal_git_icon.svg',
+    icon1: './assets/icons/modal_export_icon.svg',
+    specialization: 'Back End Dev',
+    specialization1: 'Back End Dev',
+    year: '2015',
+    year1: '2015',
+    description:
+      'A daily selection of privately <br class='must-ignor'>personalized reads;no accounts or <br class='must-ignor'>sign-ups required.',
+    description1:
+      'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent',
+    technologies: {
+      tech1: 'html',
+      tech4: 'Ruby on rials',

```

Comment:
- [ ] Your `projects` array of objects has incorrect single quotes on HTML strings. This is the reason the project details are not being displayed at the moment. See a better structured one below and fix the rest according to it:
```js
const projects = [
  {
    id: 'card1',
    title: 'Tonic',
    company: 'CANOPY',
    icon: './assets/icons/modal_git_icon.svg',
    specialization: 'Back End Dev',
    year: '2015',
    description:
      'A daily selection of privately <br class="must-ignore">personalized reads;no accounts or <br class="must-ignore">sign-ups required.',
    technologies: {
      tech1: 'html',
      tech4: 'Ruby on Rails',
      tech2: 'css',
      tech3: 'javaScript',
    },
    details: 'See project',
    imageMobile: './assets/img/cards/card1.png',
    imageDesktop: './assets/img/cards/card1.png',
    live: 'https://wilfriedm01.github.io/',
    source: 'https://github.com/WilfriedM01/portfolio-project.git',
  },
  // ... other projects ...
];

``

Reply:
Hi @codepantha 

Thank you for the feedback, I got that bug by trying to fix linter errors :smile: 
Now I have fixed and everything is okay :+1: 

Thank you